### DISCUSSION TITLE
Show latest podcasts in Android Auto

### DISCUSSION THREAD

**Discussion by osimon72** [advplyr/audiobookshelf-app#1414] [Ideas]
Hi,
I love the ABS and the Android app. I especially use the lates episodes tab in the app.
Unfortunately this is not available in Android Auto. Only the podcasts are listed. 
I would love to have the "latest episodes" in Android Auto for Podcasts, since it removes distraction during driving, easily selecting latest podcasts.

Thanks

**Comment by bobloadmire** [comment]
yes please. I can't resume whatever I was last playing easily. I have to navigate to whatever it was I was listening to last manually, really annoying.

**Reply by huggenknubbel** [reply]
you can do this on your smartpone and start from there. It will Play in android Auto. But i agree, thats not the idea of android auto.
